migration/v0.24.x-v0.25.x/add-support-message.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+sc_config="${CK8S_CONFIG_PATH}/sc-config.yaml"
+support_file="../template/common/dashboard-support-string.yaml"
```

Comment:
Could you maybe add the file to the migration folder in this repo instead? Or do we need the file later in the other repo?

Reply:
Changed it now to grab the value from the templates via yq4 instead, what do you think?

Reply:
I have not learned the yq4 syntax yet, but if it works then that seems good.
Could you also add in the `upgrade-apps` doc that one should install yq4 as well?

So you will move it here?

Reply:
that other file isn't necessary anymore, it just contained the yaml values for the `welcomingDashboards[]` keys which are also set in the templates. Sure I will add it as a note

Reply:
Not sure we are talking about the same file. I'm referring to `support_file="../template/common/dashboard-support-string.yaml"` that seem to be in another repo.

Reply:
I made that file in `ck8s-ops/environments/templates/common/` but it isn't needed anymore

Reply:
Sorry I missed that you changed which file you are looking at.
I'm still not sure that i 100% like referring to files outside this repo, but for this specific thing it might be fine.